armi/settings/fwSettings/globalSettings.py

```diff
@@ -267,14 +272,18 @@ def defineSettings() -> List[setting.Setting]:
             description="Number of depletion substeps, n, in one cycle. Note: There "
             "will be n+1 time nodes and the burnup step time will be computed as cycle "
             "length/n.",
+            schema=vol.Range(min=0),
         ),
         setting.Setting(
             CONF_BETA,
             default=None,
-            label="Delayed neutron fraction",
-            description="Set individual precursor group delayed neutron fractions.",
+            label="Delayed Neutron Fraction",
```

Comment:
We use setting the beta and lambda values to zero to determine if we should use the values directly or expect them to be calculated. I would recommend allowing zero values still.

Reply:
Done.